README.md

```diff
@@ -283,26 +283,11 @@ A FRAME pallet is compromised of a number of blockchain primitives:
 First, install [Docker](https://docs.docker.com/get-docker/) and
 [Docker Compose](https://docs.docker.com/compose/install/).
 
-Then run the following command to start a single node development chain.
+Then run the following commands to start a single node development chain.
 
 ```bash
-./scripts/docker_run.sh
-```
-
-This command will firstly compile your code, and then start a local development network. You can
-also replace the default command
-(`cargo build --release && ./target/release/node-template --dev --ws-external`)
-by appending your own. A few useful ones are as follow.
-
-```bash
-# Run Substrate node without re-compiling
-./scripts/docker_run.sh ./target/release/node-template --dev --ws-external
-
-# Purge the local dev chain
-./scripts/docker_run.sh ./target/release/node-template purge-chain --dev
-
-# Check whether the code is compilable
-./scripts/docker_run.sh cargo check
+docker compose down --volumes
+sudo ./scripts/run/subtensor.sh -e docker --network mainnet --node-type lite
```

Comment:
So I believe we should add a `-b` flag explicitly. I have seen at least two cases where the users ran this command it came back with "node-subtensor command not found". This meant that the binary doesn't exist. Is this script supposed to build the "node-subtensor" without exception or only under some conditions? See a bit more detail in this Discord thread: https://discord.com/channels/799672011265015819/1176889736636407808/1219454860508794911